api-consumption/src/main/java/org/zalando/nakadi/service/subscription/state/StreamingState.java

```diff
@@ -109,6 +112,7 @@ public void onEnter() {
         addTask(this::initializeStream);
         addTask(this::pollDataFromKafka);
         scheduleTask(this::checkBatchTimeouts, getParameters().batchTimeoutMillis, TimeUnit.MILLISECONDS);
+        scheduleTask(this::autocommitPeriodically, AUTOCOMMIT_INTERVAL_SECONDS, TimeUnit.SECONDS);
```

Comment:
why are we sure that 5 seconds is enough in terms of speed of consumption and resource usage ?

Reply:
That was my best guess, I have no other excuses